samplers.go

```diff
@@ -19,6 +19,7 @@ type DDMetric struct {
 	Tags       []string      `json:"tags,omitempty"`
 	MetricType string        `json:"type"`
 	Hostname   string        `json:"host"`
+	DeviceName string        `json:"device_name"`
```

Comment:
[Seems to be](https://github.com/DataDog/dd-agent/blob/5ebd8f33daaabd7daab225a6d3a30f229a838711/aggregator.py#L614)!